tests/devices/unit_tests/test_xspress3mini_OA.py

```diff
@@ -0,0 +1,45 @@
+import asyncio
+
+import pytest
+from bluesky import plan_stubs as bps
+from bluesky.run_engine import RunEngine
+from ophyd.status import Status
+from ophyd_async.core import DeviceCollector, set_sim_value
+
+from dodal.devices.ophyd_async_xspress3_mini.xspress3_mini_oa import (
+    AcquireState,
+    DetectorState,
+    Xspress3Mini,
+)
+
+
+def get_bad_status() -> Status:
+    status = Status("get_bad_status")
+    status.set_exception(Exception)
+    return status
+
+
+@pytest.fixture
+async def fake_xspress3mini(prefix: str = "huh") -> Xspress3Mini:
```

Comment:
Fake or mock?
@callumforrester

Reply:
With the latest `ophyd-asyn`, I think we should try to use `mock` universally

Reply:
@Relm-Arrowny @stan-dot Now that #525 is merged you can just use the mock backend, and you have to actually or the PR will break